src/gfx/shaders/Uber_frag.glsl

```diff
@@ -425,7 +426,7 @@ float unpackRGBAToDepth( const in vec4 v ) {
   	#pragma unroll_loop
```

Comment:
Unfortunately, we can't use _unroll_loop_ here. Please use old _define_ approach

Reply:
I had small research and recognize that unroll_loop have warnings in case of using *[i] in function parameters. But if assign *[i] to a new variable and then use this new variable in function parameters there will not cause warnings

Reply:
Sorry, there was a mistake in my previous answer.
Real cause is that there are some requirements for using unrol_loop in code. One of them: loop must follow a very strict pattern.
Here there are more details: https://threejs.org/docs/index.html#api/en/materials/ShaderMaterial